src/components/shared/Slider/Marks.tsx

```diff
@@ -0,0 +1,354 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { StyleProp, StyleSheet, TouchableWithoutFeedback, ViewStyle } from 'react-native';
+
+import styled from 'styled-components/native';
+
+interface MarkPositionerType {
+  width: number;
+  position: number;
+}
+
+interface Props {
+  testID?: string;
+  sliderWidth: number;
+  style?: StyleProp<ViewStyle>;
+  mark?: React.ReactElement;
+  customMarkWidth?: number;
+  step?: number;
+  pixelsPerStep?: number;
+  markCount?: number;
```

Comment:
Do we really need `pixelsPerStep`, `markCount` to display marks?
I think if just `step` can cover all cases we don't need to have them.
They might make code complex.

Reply:
They are just for customizing props. I agree that they are not necessary to construct the component. But `@dooboo-ui/native` is a library, not an application. I think we can provide more flexible props to customize in various use-cases.

Reply:
Yes, I got it. Flexible props are important.  I just wanted to say simplicity also important.
Could you explain what `pixcelsPerStep` is for?

Reply:
I tried to implement `pixelsPerStep` is the actual pixels between marks, not value at the start of implementation.

I'm thinking the reason again why ryou are saying we don't need `pixelsPerStep`.
Is it because users can calculate the actual gap by `width` of Slider? hmm..
If so, I should modify the logic related to the prop. May I modify it in the next PR for separating the purposes of PR?

Reply:
@YongPilMoon I'll fix them in the next PR as well! Thank you!